### ISSUE TITLE
Limit the notifications notice to three minor versions

### ISSUE THREAD

**Issue by xristos3490** [Automattic/woocommerce-subscriptions-core#715]
## Description

Follow-up on https://github.com/Automattic/woocommerce-subscriptions-core/pull/713

The goal is to prevent the notice from appearing after three minor plugin versions, starting with version WCS v6.9.0.

## Product impact

- [ ] Does this feature affect WooCommerce Subscriptions? yes/no/tbc, add issue ref
- [ ] Does this feature affect WooCommerce Payments? yes/no/tbc, add issue ref

**Comment by alefesouza** [comment]
@mattallan as we already had more than 3 minor versions after this was released, does it make sense to open a PR simply removing this notification?

**Comment by mattallan** [comment]
@alefesouza yeah, I think it's fine for us to simply remove this admin notice now 👍
